src/main/java/hu/eszter/bokkon/service/Util.java

```diff
@@ -35,38 +63,46 @@ public static String getAnsiBrightGreen() {
      *
      * @param possibleChanges is a map data structure storing the possible exchanges between 2 animal stocks
      */
-    public static void printPossibleChangesMap(Map<Animal, Map<Animal, Double>> possibleChanges) {
+    public static void displayPossibleExchanges(Map<Animal, Map<Animal, Double>> possibleChanges) {
         int line = 1;
-        for (Animal exchangeAnimal : possibleChanges.keySet()) {
-            String animalName = exchangeAnimal.getClass().getSimpleName();
-            Map<Animal, Double> actMap = possibleChanges.get(exchangeAnimal);
+        for (Animal actFarmerAnimal : possibleChanges.keySet()) {
+            String animalName = actFarmerAnimal.toString().toLowerCase();
```

Comment:
Optional: You could add a 'displayName' String property to the enum (similarly to AnimalType), and then you don't have to write toString().toLowerCase() this many times